test/unit/custom_message_setting_test.rb

```diff
@@ -62,6 +75,22 @@ def test_update_with_custom_messages_yaml_if_yaml_is_invalid
     assert_equal "(<unknown>): could not find expected ':' while scanning a simple key at line 4 column 1", @custom_message_setting.errors[:base].first
   end
 
+  def test_toggle_enabled!
+    assert_equal 'Home1', l(:label_home)
```

Comment:
It is easy to understand by inserting "asset @custom_message_setting.enabled?" .